apps/full-site-editing/package.json

```diff
@@ -32,17 +32,17 @@
 		"build": "rimraf dist && npm-run-all --parallel build:*"
 	},
 	"dependencies": {
-		"@wordpress/api-fetch": "^3.1.2",
-		"@wordpress/blocks": "^6.2.3",
-		"@wordpress/components": "^7.3.0",
-		"@wordpress/compose": "^3.2.0",
+		"@wordpress/api-fetch": "3.2.0",
+		"@wordpress/blocks": "6.3.0",
+		"@wordpress/components": "7.4.0",
+		"@wordpress/compose": "3.3.0",
 		"@wordpress/data": "^4.4.0",
-		"@wordpress/edit-post": "^3.3.4",
+		"@wordpress/edit-post": "3.3.6",
```

Comment:
Renovate uses exact versions whenever it updates a dependency? Do we want that for all packages?

Probably doesn't matter at all for `/apps`, which we're building from here. It is something to consider as we publish more `/packages`.